### PR TITLE
The creation of resources still uses hardcoded values, but at least we have warnings

### LINKED ISSUE #1553: The creation of resources uses hardcoded values
**Description**

When we create a BTP Service Binding instance we use  /v1alpha when /v1 is already available

**Expected result**

We should all the time user the latest version that is available in the cluster. This seems to be a general issue that is not targeted towards BTP Service only.

**Actual result**

**Steps to reproduce**

**Troubleshooting**

image
image

### PR DESCRIPTION
**Pull request by Wawrzyn321**
**Testing**

You can check it out on https://warn.kmain.hasselhoff.shoot.canary.k8s-hana.ondemand.com

I've already makes sure that Sentry messages will be there:
image

We even have alerts!
image

**Related issue(s)**

Connected #1566 

Closes #1553

### CODE REVIEW

**Review by dariadomagala — APPROVED** [review]

File: core/src/i18n/en.yaml [hunk 1]
```diff
@@ -1621,6 +1621,8 @@ node-details:
   warnings: Warnings
 nodes:
   name_singular: Node
+  statuses:
+    ready: Ready
```
[the remark this answers is not in the document]
Comment (Wawrzyn321) [reply core/src/i18n/en.yaml:1625]: Good point.

**Review by dariadomagala — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -17,6 +17,11 @@ Busola project consists of the following UI projects:
 - [`npm`](https://www.npmjs.com/): >= 8.1.2
 - [`node`](https://nodejs.org/en/): >= 16.13.2
 
+Busola supports:
+
+- [Kyma](https://kyma-project.io/): >= 2.4.2
```
Comment (klaudiagrz) [inline README.md:22]:
```suggestion
- [Kyma](https://kyma-project.io/) in version 2.4.2 or higher
```

File: README.md [hunk 3]
```diff
@@ -17,6 +17,11 @@ Busola project consists of the following UI projects:
...
+- [Istio](https://istio.io/): resource in version `v1beta1`.
```
Comment (klaudiagrz) [inline README.md:23]:
```suggestion
- [Istio](https://istio.io/) in version `v1beta1`
```

**Review by akucharska — APPROVED** [review]

**Pull request merged**